guide/yaml/winrm/index.md

```diff
@@ -312,7 +312,7 @@ The script works for most of the Windows images out there version 2008 and later
 
 Please ensure that Brooklyn's changes are compatible with your organisation's security policy.
 
-Check Microsoft Documentation for more information about [Negotiate authenticate mechanism on technet.microsoft.com](https://msdn.microsoft.com/en-us/library/windows/desktop/aa378748\(v=vs.85\).aspx)
+Check Microsoft Documentation for more information about [Negotiate authenticate mechanism on technet.microsoft.com](https://msdn.microsoft.com/en-us/library/aa384295(v=vs.85).aspx)
```

Comment:
When I test locally with `jekyll serve`, it doesn't need the backslash before the bracket - and gives the wrong url with the backslash. For example, the old link points at
https://msdn.microsoft.com/en-us/library/windows/desktop/aa378748/(v=vs.85/).aspx

Looking again, that is all that was wrong with the link (making it broken in the final docs - see http://brooklyn.apache.org/v/latest/yaml/winrm/index.html).

I'll change it back to the original link, with the backslashes removed.